pkg/operators/machine-api-operator.go

```diff
@@ -24,289 +24,298 @@ var (
 	maoManagedDeployment = "machine-api-controllers"
 )
 
-var _ = Describe("[Feature:Operators][Disruptive] Machine API operator deployment should", func() {
-	defer GinkgoRecover()
+var _ = Describe(
+	"[Serial][Feature:Operators][Disruptive] Machine API operator deployment should",
+	Label(framework.LabelDisruptive, framework.LabelOperators, framework.LabelMachines),
+	Serial,
+	func() {
+		defer GinkgoRecover()
```

Comment:
You don't actually need this, though I know it was already there

Reply:
Removed